index.js

```diff
@@ -0,0 +1,79 @@
+let classAdd = document.querySelector("#classAdd")
+let addClass = document.querySelector("#addClass")
+let studentEnroll = document.querySelector("#studentEnroll")
+let addStudent = document.querySelector("#addStudent")
+let list = document.querySelector("#list")
+let listStudents = document.querySelector("#listStudents")
+addClass.addEventListener("submit",async (e)=>{
+    e.preventDefault()
+            let classes = document.querySelector("#className")
+            let teacher = document.querySelector("#teacherName")
+            let classesInput = classes.value
+            let teacherInput = teacher.value
+            classesInput.value = ""
+            teacherInput.value = ""
+            let p = document.createElement("p")
+            try{
+
+                let host = await axios.post(`http://localhost:4000/class`, {teacher: teacherInput, class: classesInput})
+                debugger
+                p.innerText = host.data
+                classAdd.appendChild(p)
+            }catch(error){
+                
```

Comment:
you might want to console.log the error for yourself, so you could see what doesn't work (if it doesn't)